File: monte-carlo-simulation-julia/Plot_D.jl
using PyPlot
include("monte_carlo.jl")
include("sampleThreshold.jl")
k = logspace(-5,2,200)
(L,M) = monte_carlo(sampleThreshold, k , 1e5)
D = L./M
N = 1./M
plot(N,D)

File: MCsim-Mod1/MC-sim-matlab-Mod1/Plot_D.m
k = linspace(0,100,200);
[L,M] = monte_carlo(@sampleRandom, k , 1e5);
D = L./M;
N = 1./M;

save D D
save alpha N
save('D:\\Documents\\GitHub\\remote-estimation\\monte-carlo\\D.mat','D')
save('D:\\Documents\\GitHub\\remote-estimation\\monte-carlo\\alpha.mat','N')
figure;
plot(N,D);
title('Distortion');
xlabel('alpha');
ylabel('D');
